src/amazonqGumby/telemetry/codeTransformTelemetry.ts

```diff
@@ -30,20 +30,22 @@ export enum StartActionPositions {
 
 export const logCodeTransformInitiatedMetric = (source: string): void => {
     const commonMetrics = {
-        codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
+        codeTransformSessionId: `${codeTransformTelemetryState.getSessionId()}`,
```

Comment:
Dump the string literal and the ${. Just log `codeTransformTelemetryState.getSessionId()`